index.html

```diff
@@ -25,6 +26,13 @@
     <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
       <img src="/assets/img/homepage/Static_Header.svg" alt="BCDA logo" class="ds-l-sm-col--6 ds-l-md-col--6 ds-l-lg-col--12">
     </div>
+    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
```

Comment:
I think the column sizes for the three columns need to shrink from 6 --> 4:

```suggestion
    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--4 ds-u-text-align--center">
```

Right now it pops down into a new row.

<img width="1421" alt="Screen Shot 2023-08-03 at 5 13 04 PM" src="https://github.com/CMSgov/bcda-static-site/assets/2308368/5babd35d-af75-4532-832e-53a02c0550cf">